src/components/Auth/LoginComponent.jsx

```diff
@@ -30,13 +32,24 @@ class LoginComponent extends Component {
         window.location.replace('/');
```

Comment:
You could use redirect from `react router dom`

Reply:
https://reacttraining.com/react-router/web/api/Redirect

Reply:
There is also the option of using 
```
this.props.history.push('/')
```